src/Buckle/Repl/BelteRepl.cs

```diff
@@ -371,22 +371,36 @@ internal enum Page {
     }
 
     [MetaCommand("dump", "Show contents of symbol <name>")]
-    private void EvaluateDump(string name) {
+    private void EvaluateDump(string signature) {
```

Comment:
make sure that the EmitTree function ignores struct bodies if the type has dimensions